### DISCUSSION TITLE
Allow Switches to be included in Light Groups

### DISCUSSION THREAD

**Discussion by dougmaitelli** [home-assistant/frontend#12941] [Other feature requests] [answered]
I have a situation where I need to group a switch and a light. I want to create a group that when ON will turn on both the switch and the light but will also be able to control the color of the light.
So basically allow the inclusion of switches in light groups.

This was possible before the introduction of helpers in the UI and is even suggested as a solution in the forums: https://community.home-assistant.io/t/combining-lights-and-switches/347284

**Comment by frenck** [comment, marked as answer]
This is not a frontend concern. Please raise a feature request on the community forum instead.
